modules/asset/ui/src/_locales.scss

```diff
@@ -0,0 +1,38 @@
+.locales {
+  position: absolute;
+  text-align: right;
+  top: 0;
+  right: 20px;
+}
+
+.locales__toggler {
+  cursor: pointer;
+
+  &[aria-expanded="true"] {
+    border-radius: 10px 10px 0px;
+  }
+}
+
+.locales__list {
```

Comment:
The bottom-left corner looks odd. Also that the text is not aligned the same way. Let me know if this is from a design.

![Screen Shot 2021-11-29 at 12 57 49 PM](https://user-images.githubusercontent.com/1155984/143926688-d8d4c166-eebc-4cac-bf1e-1f5661de93c0.png)